### DISCUSSION TITLE
Tactile paving on crossings mapped with two nodes (e.g. nearby roundabouts)

### DISCUSSION THREAD

**Discussion by ivanbranco** [streetcomplete/StreetComplete#5113] [Q&A]
Hi, it's me overthinking SC quests again.

In cases like this ([OSM location](https://www.openstreetmap.org/way/935343393/history)):
![photo_2023-07-02_13-25-19](https://github.com/streetcomplete/StreetComplete/assets/94002623/7b593a69-7409-4495-9545-4f55d998d708)

where a crossing is mapped with two nodes, should I reply that both nodes have tactile paving on both sides (tactile_paving=yes), or that they have only one side (tactile_paving=incorrect?) since between them there's no additional tactile paving?

**Comment by Helium314** [comment]
I answer `yes` in such cases. This kind of crossing is perceived as a single crossing,, and only split because of lane micromapping.
Since the crossing doesn't have an island I definitely wouldn't expect tactile paving in the center.
If an answer would allow specifying the side instead of tagging `incorrect`, I'd go with the micromapping though and specify the side.

One may argue that technically `yes` for both is incorrect, but imo it's as correct as mapping two pedestrian crossings here.
